chain/gen/genesis/genesis.go

```diff
@@ -375,11 +419,21 @@ func VerifyPreSealedData(ctx context.Context, cs *store.ChainStore, stateroot ci
 		return cid.Undef, xerrors.Errorf("failed to create NewVM: %w", err)
 	}
 
-	for _, m := range template.Miners {
+	for mi, m := range template.Miners {
+
+		/*
+			// Add the miner to the market actor's balance table
```

Comment:
Where does this happen instead now?

Reply:
Update: Wasn't happening earlier either (was erroring), so probably not needed because market balance table will assume it has 0?